src/data/detail-page/Descriptions/backendDescription.ts

```diff
@@ -0,0 +1,2 @@
+// TODO: Add longer description
```

Comment:
Let's ensure we have uniform file and directory names across the project. 
Directory names should follow kebab-case (as in the rest of the project). Files like GraphInterfaces or LearnInterfaces are not jsx components, so they should be camelCase. Project manager is two words and the file name should reflect it: `projectManagerGraph.ts`.
Also, if we our directory name is e.g. descriptions, mayeb we don't need to use the `description` suffix for all the files inside this directory (that is instead of `backendDescription` have simply `backend)`? This would make the directory content far more readable! Finally, it's good practice to ensure the exported const name matches the file name. So in a file `backend.ts` we'd have `export const backend = ...`.
![image](https://user-images.githubusercontent.com/8350925/113033177-07ad1080-9191-11eb-8c69-20a97341acee.png)

Reply:
Problem is that I import file by by for component by it route Id. Route id for project menager, devOps are lover case, so they would not import in components if they name will be camel case.